### ISSUE TITLE
Add ChowDSP ChowTape

### ISSUE THREAD

**Issue by jatinchowdhury18** [surge-synthesizer/surge#3585]
Add [ChowTape](https://github.com/jatinchowdhury18/AnalogTapeModel) to the Surge effects rack. As mentioned in #1694 

ChowTape is a an analog tape simulation, based on a corresponding [academic paper](http://dafx2019.bcu.ac.uk/papers/DAFx2019_paper_3.pdf).

The original has many parameters, so I think there's two ways to deal with this. One option is to split the effect into several mini-effects, for example Tape Hysteresis, Tape Loss, Tape Wow/Flutter. The other option is to hardcode some of the parameters, and only expose the most important/noticeable ones.

If we were to go with the second option, here's the parameters I think I should include:
Drive
Saturation
Bias
Tape Speed
Spacing
Thickness
Gap
Wow Rate
Wow Depth
Flutter Rate
Flutter Depth

**Comment by mkruselj** [comment]
We will be increasing the parameter count in 1.9 (#3257) so more params could be supported.

Personally I think we should have it all in one effect. The choices we have to make is which processing mode to use (I see there's options for Runge-Kutta versus Newton-Raphson etc.), and maybe condense other less important parameters into "mode" presets, perhaps?

**Comment by jatinchowdhury18** [comment]
Hi all, I've started working for including ChowTape, on [this branch](https://github.com/Chowdhury-DSP/surge/tree/chowdsp-tape1). So far, I've only implemented the hysteresis engine (the most important part imo). For now I'm using a Newton-Raphson solver, with 3 iterations, sort of a compromise between accuracy and efficiency. As always, testing and feedback is much appreciated!

Up next, I'm hoping to implement the loss filters, though I'm expecting to have to fight to minimise latency.
